munch-match/.env-template

```
@@ -1 +1,2 @@
-REACT_APP_GOOGLE_API_KEY = 
\ No newline at end of file
+REACT_APP_GOOGLE_API_KEY = AIzaSyBTDgz5uQ2c_lxNFzUGqudqF7y6e31JCzw
+REACT_APP_SPOONACULAR_API_KEY = f3a6e7d0c32c4e25baf2aee268fa341a
```

```suggestion
REACT_APP_SPOONACULAR_API_KEY =
```

Just for the reason mentioned in the other comment